assets/api-swagger/swagger.json

```diff
@@ -914,6 +914,228 @@
+                  "offset": 2,
+                  "limit": 10,
+                  "job_count": 3,
+                  "total_job_count": 5,
+                  "jobs": [
+                    {
+                      "job_id": 12,
+                      "status": "finished"
+                    },
+                    {
+                      "job_id": 13,
+                      "status": "failed"
+                    },
+                    {
+                      "job_id": 14,
+                      "status": "in_progress"
+                    }
+                  ]
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Missing account ID header.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorMessage"
+                },
+                "example": {
+                  "status": 400,
+                  "type": "about:blank",
+                  "detail": "Missing account ID header",
+                  "title": "Bad Request"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "An error occurred.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorMessage"
+                },
+                "example": {
+                  "status": 500,
+                  "type": "about:blank",
+                  "detail": "Internal Server Error",
+                  "title": "An error occurred"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/merge-profile/{job_id}": {
+      "get": {
+        "tags": [
+          "Profiles"
+        ],
+        "summary": "Get the status of a specific merge job.",
+        "description": "Get the status of a specific merge job.",
+        "parameters": [
+          {
+            "name": "Ometria-Account-Id",
+            "in": "header",
+            "description": "ID for the account in which to search for the profile.",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "job_id",
+            "in": "path",
+            "description": "The id of the merge job.",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The statuses of the merge jobs.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/MergeProfileJobStatus"
+                },
+                "example": {
+                  "job_id": 1234,
+                  "status": "finished"
+                }
+              }
+            }
+          },
+          "400": {
```

Comment:
Same question regarding the response with status code 403.

Reply:
same answer